src/com/lingoport/demo/java/ExamplePanel.java

```diff
@@ -96,9 +96,10 @@ private void initialize() {
 
     public void createButtons() {
 
-        ImageIcon nextButtonIcon = ImageHelper.createImageIcon("images/next.jpg");
-        nextButton_ = new JButton( nextButtonIcon);
-        nextButton_.setToolTipText("Next & Clear");
+        ImageIcon nextButtonIcon3 = ImageHelper.createImageIcon("images/next.jpg");
+        ImageIcon nextButtonIcon2 = ImageHelper.createImageIcon("images/next.jpg");
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused "nextButtonIcon2" local variable. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://localhost:9010/coding_rules#rule_key=squid%3AS1481)